example/lib/main.dart

```diff
@@ -374,11 +374,22 @@ class _MyAppState extends State<MyApp> implements RoomHandler {
   }
 
   @override
-  Future<void> addRemoteStream(MediaStream stream) async {
+  Future<void> addRemoteStream(int trackId, MediaStream stream) async {
     await _rendererInitialized;
     _remoteRenderer.srcObject = stream;
```

Comment:
@vsomayaji we talked about that, currently is expected to have multiple renderers but when talking with @ipainchaud we think it will more cleaner only to have one renderer and attach all tracks to it, do not know if this is the right time for that refactor

Reply:
This is not what I did @smunozp . I have retained the original approach with multiple renderers.